pkg/apis/v1beta1/nodepool.go

```diff
@@ -135,6 +136,10 @@ func (in *NodePool) Hash() string {
 	})))
 }
 
+func (in *NodePool) Validate() (errs *apis.FieldError) {
```

Comment:
```suggestion
func (in *NodePool) Validate() (errs error) {
```

I think we should just default to using regular errors on the path to dropping the current dependency that we have on knative